src/app/public/modules/list-view-grid/list-view-grid.component.ts

```diff
@@ -113,6 +162,12 @@ export class SkyListViewGridComponent
   @Input()
   public settingsKey: string;
 
+  @Output()
+  public rowDeleteCancel = new EventEmitter<SkyGridRowDeleteCancelArgs>();
```

Comment:
I noticed that for this `Output`, we're using the Grid's args type, but for the `Input`s we've created duplicate View-Grid types (e.g., `SkyListViewGridMessage`). Is there anything preventing us from just using the Grid's types, and not duplicating the work for View-Grids?

Reply:
This was deliberate - it all comes down to tiptoeing around the list-builder complexity.

We didn't want to expose the vanilla grid's `SkyGridMessageType` because it also had properties to select and deselect rows on the grid. Exposing those properties might confuse consumers using the list-view-grid because unlike the vanilla grid, list-builder handles selections. If we exposed `SkyGridMessageType`, then you could have two different paths to make selections on the grid and get into some serious sync issues.

Reply:
Could we create a duplicate `SkyListViewGridRowDeleteCancelArgs` (instead of passing through `SkyGridRowDeleteCancelArgs`), then?

Reply:
Good idea. This is done.